contracts/protocol/pool/Pool.sol

```diff
@@ -992,10 +992,11 @@ contract Pool is ReentrancyGuard, VersionedInitializable, PoolStorage, IPool {
     /// @inheritdoc IPool
     function rescueTokens(
         address token,
+        DataTypes.AssetType assetType,
```

Comment:
could you put assetType before token which makes more sense. and this may breaks some test cases in paraspace-periphery

Reply:
have adjusted the order of the params. There are no breaks for test cases, And I have added new unit tests for rescueTokens function.